lte/gateway/c/oai/tasks/sctp/sctp_itti_messaging.c

```diff
@@ -69,16 +100,41 @@ int sctp_itti_send_new_message_ind(
   SCTP_DATA_IND(msg).assoc_id = assoc_id;
 
   STOLEN_REF* payload = NULL;
-
-  return send_msg_to_task(&sctp_task_zmq_ctx, TASK_S1AP, msg);
+  if (ppid == S1AP) {
+    OAILOG_DEBUG(
+        LOG_SCTP, "ppid match S1AP in sctp_itti_send_new_message_ind ");
+    return send_msg_to_task(&sctp_task_zmq_ctx, TASK_S1AP, msg);
+  } else if (ppid == NGAP) {
+    OAILOG_DEBUG(
+        LOG_SCTP, "ppid match NGAP in sctp_itti_send_new_message_ind ");
+    return send_msg_to_task(&sctp_task_zmq_ctx, TASK_NGAP, msg);
+  } else {
+    OAILOG_ERROR(
+        LOG_SCTP, "ppid: %d not matching in sctp_itti_send_new_message_ind ",
+        ppid);
+    return 0;
```

Comment:
Use the macros RETURNok or RETURNerror instead of numerals in return code

Reply:
ok